conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -0,0 +1,203 @@
+import os
+
+# FIXME: imports from conan namespace
+import textwrap
+
+from conans.client.build.compiler_flags import rpath_flags, format_frameworks, format_framework_paths
```

Comment:
This shouldn't be using anything from the old scope, but from ``from conan.tools``. If necessary bring a copy of those to this gnu. Make sure they are not already there, maybe in ``Autotools``.

Reply:
Indeed! Changing it...